exercises/concept/ellens-alien-game/.docs/instructions.md

```diff
@@ -0,0 +1,116 @@
+# Instructions
+
+Ellen is making a game where the player has to fight aliens.
+She has just learned about Object Oriented Programming (OOP) and is eager to take advantage of what this paradigm has to offer.
+
+To Ellen's delight, you have offered to help and she has given you the task of programming the aliens that the player has to fight.
+
+## 1. Create the Alien Class
+
+Define the Alien class so that it accepts an x and a y position, and puts them into `x` and `y` variables.
+Every alien starts off with a health of 3, so initialize the `health` variable too.
```

Comment:
Yeah, I don't know either.  When I was writing it up, I didn't want to be _so_ explicit about _what_ to do that I might accidentally stumble into telling the user _how_ to do it.  I do hint at the x and y coords being instance variables in the hints file, though maybe not explicitly enough.

The whole `x_coordinate` and `y_coordinate` thing is a good idea.  I didn't know that there was a single-letter-variable rule (whoops!), but I do see that explicitly specifying them as coordinates is probably going to help.  It will mean some finding/replacing, though 😆 

I like the new line 11 though.